client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/packages-step/package-info.js

```diff
@@ -122,12 +123,14 @@ const PackageInfo = ( props ) => {
 
 	const renderPackageSelect = () => {
 		if ( isIndividualPackage ) {
+			const dimensionsClass = classNames( { 'is-error': pckgErrors.dimensions } );
 			return ( <div>
 				<div className="packages-step__package-items-header">
 					<FormLegend>{ translate( 'Individually Shipped Item' ) }</FormLegend>
 				</div>
 				<span className="packages-step__package-item-description">{ translate( 'Item Dimensions' ) } - </span>
```

Comment:
Hi! I've found a possible matching string that has already been translated **103** times:
`translate( 'Dimensions' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/wcs-1239-dimensions)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).